tests/dummy/app/app.js

```diff
@@ -6,6 +6,7 @@ import config from './config/environment';
 let App;
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
+Ember.run.backburner.DEBUG = true;
```

Comment:
This setting might be helpful, but the docs indicate that it slows down even normal development substantially. Would you be ok with keeping it turned off for the dummy app, unless explicitly needed?

https://guides.emberjs.com/v2.4.0/configuring-ember/debugging/#toc_errors-within-code-ember-run-later-code-a-href-https-github-com-ebryn-backburner-js-backburner-js-a